kura/org.eclipse.kura.net.admin/META-INF/MANIFEST.MF

```diff
@@ -14,7 +14,7 @@ Import-Package: javax.microedition.io,
  org.eclipse.kura.core.configuration.metatype;version="[1.0,2.0)",
  org.eclipse.kura.core.linux.executor;version="[1.0,2.0)",
  org.eclipse.kura.core.linux.util;version="[1.0,2.0)",
- org.eclipse.kura.core.net;version="[1.0,2.0)",
+ org.eclipse.kura.core.net;version="[1.0,2.1)",
```

Comment:
If the net APIs have moved to 2.x I believe this range needs to be changed to [2.0,3.0). If not, the range can stay the same, right?

Reply:
It probably comes from a merge. I will move it back to 2.0.